### PR TITLE
Apartment show

### PR DESCRIPTION
**Pull request by JohnRucker4**

### CODE REVIEW

File: app/javascript/components/pages/ApartmentShow.js [hunk 1]
```diff
@@ -7,11 +7,13 @@ import {NavLink} from "react-router-dom"
...
     const { id } = useParams()
     const navigate = useNavigate()
-    let selectedApartment = apartments.find(apartment => apartment.id === +id)
 
-    const handleSubmit = () => {
-      navigate("/apartmentindex")
-    }
+    let selectedApartment = apartments?.find(apartment => apartment.id === +id)
+
+    // const handleSubmit = () => {
```
Comment (Genemichael25) [inline app/javascript/components/pages/ApartmentShow.js]: We don't want code that is commented out on main.

File: app/javascript/components/pages/ApartmentShow.js [hunk 2]
```diff
@@ -27,41 +29,49 @@ const ApartmentShow = ({apartments}) => {
         />
         <CardBody>
           <CardTitle>
-            {selectedApartment.address}
+
```
Comment (Genemichael25) [inline app/javascript/components/pages/ApartmentShow.js]: Can you also remove extra lines with no code please.

**Review by Genemichael25 — CHANGES_REQUESTED** [review]

File: app/javascript/components/pages/ApartmentShow.js (hunk 2 above)
Comment (JohnRucker4) [reply app/javascript/components/pages/ApartmentShow.js]: We removed the empty lines and commented code.  Thanks, Gene

File: app/javascript/components/pages/ApartmentShow.js (hunk 1 above)
Comment (JohnRucker4) [reply app/javascript/components/pages/ApartmentShow.js]: Done and done

File: app/javascript/components/pages/ApartmentShow.js [hunk 3]
```diff
@@ -27,44 +24,50 @@ const ApartmentShow = ({apartments}) => {
...
+      to={`/apartmentedit/${selectedApartment?.id}`}>
+
           Edit Apartment
       </NavLink>
       </Button>
       <Button>
       <NavLink 
       to={`/apartmentindex`}>
-          Back to Apartment Index
+
```
Comment (Genemichael25) [inline app/javascript/components/pages/ApartmentShow.js]: Still have a few blank lines, can you take care of those before you merge, thanks!

**Review by Genemichael25 — APPROVED** [review]

**Pull request merged**